### PR TITLE
adding Phase1 timing client with tracking

### PR DESCRIPTION
**Pull request by fioriNTU**
This PR adds an Online client with track reconstruction to perform Pixel timing in collisions

Also few minor things are modified:
1) initialized a bool
2) changed the ranges of residuals plots to be more useful with suboptimal alignment

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

File: DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+# here and not in the harvestng config.
+DefaultHisto.perLumiHarvesting = True
+DefaultHistoDigiCluster.perLumiHarvesting = True
+DefaultHistoSummary.perLumiHarvesting = True
+DefaultHistoTrack.perLumiHarvesting = True
+
+
+# Pixel Digi Monitoring
+from DQM.SiPixelPhase1Digis.SiPixelPhase1Digis_cfi import *
+SiPixelPhase1DigisAnalyzer.src = cms.InputTag("siPixelDigis") # adapt for real data
```
Comment (davidlange6) [inline DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py]: Hi @fioriNTU - this does not need to be here (as it does not change anything - if it did, then you would need to clone the module) - please remove it

File: DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+from DQM.SiPixelPhase1RawData.SiPixelPhase1RawData_cfi import *
+
+from DQM.SiPixelPhase1Common.SiPixelPhase1GeometryDebug_cfi import *
+
+from DQM.SiPixelPhase1TrackEfficiency.SiPixelPhase1TrackEfficiency_cfi import *
+#Summary maps
+#from DQM.SiPixelPhase1Summary.SiPixelPhase1Summary_cfi import *
+
+
+siPixelPhase1OnlineDQM_source = cms.Sequence(
```
Comment (davidlange6) [inline DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py]: would be great if this implementation was shared with the offline cff... (its mostly a duplication)

File: DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py (hunk 1 above)
Comment (fioriNTU) [inline DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py]: yes, thanks for noticing, I will fix

File: DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py (hunk 2 above)
Comment (fioriNTU) [inline DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py]: I would prefer to keep this completely independent by the Offline config. This piece is only used in the timing client, if someone mess it up will be transparent to the Offline

**Comment by fioriNTU** [comment]
@davidlange6 you made me spot also a trivial error, I have set the geometry back to Phase 1 also, thanks.

**Comment by fioriNTU** [comment]
@davidlange this is not going to be integrated in 91X?

**Comment by dmitrijus** [comment]
+1

File: DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py [hunk 3]
```diff
@@ -82,8 +82,8 @@
 #
   
 # To Configure Phase1 DQM for Phase0 data
-SiPixelPhase1Geometry.upgradePhase = 0
-SiPixelPhase1Geometry.onlineblock = 5
+SiPixelPhase1Geometry.upgradePhase = 1
```
Comment (davidlange6) [inline DQM/SiPixelPhase1Config/python/SiPixelPhase1OnlineDQM_Timing_cff.py]: Yuck...
lets remove this sort of flag in favor of geometry driven code..

**Comment by davidlange6** [comment]
+1

**Pull request merged**